frontend/public/style/_common.scss

```diff
@@ -48,6 +48,12 @@ dl.co-inline {
     padding-left: $pf-v6-global-gutter--md;
     padding-right: $pf-v6-global-gutter--md;
   }
+  .co-m-pane__body-item--cancel-padding {
```

Comment:
Rather than a modifier on a new block and element, it'd be better to add a modifier to the existing block.  Maybe `.pf-v6-c-tabs--offset-pane-body-padding` and then move this rule to `_overrides.scss`?

Reply:
I'm in favor of keeping it as a modifier of `co-m-pane__body` so that:
- in the future if we decide to change the padding values it's clear that we also need to update the `offset-padding` modifier
- so that the negative margins only apply if it's in a child of the element which has the padding we're trying to offset
- this modifier class need not only apply to tabs if needd

I did update the class though to `offset-padding` though since that's a better name. what do you think?